workspaces/azure-devops/plugins/scaffolder-backend-module-azure-devops/package.json

```diff
@@ -39,16 +39,16 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-plugin-api": "^0.7.0",
+    "@backstage/backend-plugin-api": "^0.8.0",
```

Comment:
Should this automation just update the backstage app and backends or also the plugins itself?

This might be a breaking change without a changeset :thinking: 

cc @backstage/community-plugins-maintainers ?

Reply:
No, this should fall to the individual plugin maintainers to manage and update based on their needs. The version bump workflow also makes that pretty easy: https://github.com/backstage/community-plugins/blob/main/docs/version-bump.md